runtime/lua/vim/shared.lua

```diff
@@ -983,4 +983,17 @@ do
   end
 end
 
+--- @generic T
+--- @param root string
+--- @param _mod T
+--- @return T
+function vim._deferload_submodules(root, _mod)
+  return setmetatable({}, {
+    __index = function(t, k)
+      t[k] = require(string.format('%s.%s', root, k))
```

Comment:
```suggestion
      local ok, ret = pcall(require, string.format('%s.%s', root, k))
      if ok then
        t[k] = ret
      end
```

Reply:
Q: What about errors upon `require`-ing? Shouldn't it throw rather than shallowing?

Reply:
Wait, why do we wanna do this again? Shouldn't bad requires fail?